src/panels/lovelace/cards/hui-markdown-card.ts

```diff
@@ -0,0 +1,86 @@
+import { html, LitElement } from "@polymer/lit-element";
+
+import "../../../components/ha-card.js";
+import "../../../components/ha-markdown.js";
+
+import { HassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+import { LovelaceCard, LovelaceConfig } from "../types.js";
+
+interface Config extends LovelaceConfig {
+  content: string;
+  title?: string;
+}
+
+export class HuiMarkdownCard extends HassLocalizeLitMixin(LitElement)
```

Comment:
Should I just extend `LitElement` instead?

Reply:
HassLocalizeLitMixin provides a `localize` function for translating text. Since you're not using that, `export class HuiMarkdownCard extends LitElement` should be enough, yes.

Reply:
Okay. I should probably create a new pull to clean up my other conversion as well then: https://github.com/home-assistant/home-assistant-polymer/pull/1801

Thanks